tox.ini

```diff
@@ -51,7 +51,7 @@ extras =
 deps =
     xdist: pytest-xdist
     cov: pytest-cov
-    jwst: jwst[test] @ git+https://github.com/spacetelescope/jwst.git
+    jwst: jwst[test] @ git+https://github.com/WilliamJamieson/jwst.git@refactor/use_stdatamodels_dq
```

Comment:
I'm assuming this is just temporary for testing and we're not going to leave it this way in the master branch?

Reply:
Correct, something like this is necessary until spacetelescope/jwst#7480 is merged because the upper pin on stdatamodels in jwst causes the changes in the stdatamodels PRs to overridden by the install of jwst.